arviz/stats/stats.py

```diff
@@ -949,6 +950,9 @@ def summary(
         Refer to documentation of az.convert_to_dataset for details
     var_names: list
         Names of variables to include in summary
+    filter_like: bool, default=False
+        Whether to interpret var_names as substrings of the real variables names. À la
+        `pandas.filter("like")`.
```

Comment:
Does pandas.filter accept regular expressions?

Reply:
Yes, if you use `regex` as kwarg instead of `like`

Reply:
Ohhh, I see now